src/app/my-info/(account)/reviews/_components/MyReviewProduct.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import { getProductDetail } from '@/api/productAPI';
+import { formatDateToString } from '@/libs/formatDateToString';
+import { ProductType } from '@/types/ProductTypes';
+import { useQuery } from '@tanstack/react-query';
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+import Link from 'next/link';
+import styles from './MyReviewProduct.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface MyReviewProductProps {
+  productId: number;
+  updatedAt: Date;
+  switchOption: string;
+}
+
+export default function MyReviewProduct({ productId, updatedAt, switchOption }: MyReviewProductProps) {
+  const { data: productDetailData } = useQuery<ProductType>({
+    queryKey: ['product-detail', productId],
+    queryFn: () => getProductDetail(productId.toString()),
+  });
+
+  if (!productDetailData) return null;
+
+  const { name, thubmnailList, categoryName } = productDetailData;
```

Comment:
```suggestion
  const { name, thubmnailList, categoryName } = productDetailData ?? {};
```
이런식으로 empty case로 보여주는게 나을거같아요